dashboard/blueprints/page/templates/descriptions.html

```diff
@@ -70,20 +72,20 @@
     <script src="{{ url_for('static', filename='js/highlight.js') }}"></script>
     <script src="{{ url_for('static', filename='js/render_vis.js') }}"></script>
     <script>
-        collapsed_char = '\u2B9E';
-        expanded_char = '\u2B9F';
+        collapsed_char = '<span class="cui-chevron-right" aria-hidden="true"></span>';
+        expanded_char = '<span class="cui-chevron-bottom" aria-hidden="true"></span>';
 
 
         expand_all = function() {
             $("div[id^=columns_]").css("display", "block");
-            $("a[id^=toggle_]").text(expanded_char);
-            $("a[id=all_toggle]").text(expanded_char);
+            $("a[id^=toggle_]").html(expanded_char);
```

Comment:
Can we expand and collapse globally using `class` rather than `id`. It would be much easier than looking for all ids with a pattern.